kong/conf_loader/init.lua

```diff
@@ -577,6 +577,11 @@ local CONF_PARSERS = {
 
   proxy_server = { typ = "string" },
   proxy_server_ssl_verify = { typ = "boolean" },
+
+  error_template_html = { typ = "string" },
+  error_template_json = { typ = "string" },
+  error_template_xml = { typ = "string" },
+  error_template_plain = { typ = "string" },
```

Comment:
@hanshuebner 
What data gets passed to these templates?

Reply:
@esatterwhite the templates are rendered by passing them _the error message_ as a string, see [here](https://github.com/Kong/kong/blob/fa90a685b5a7ac03774460a92f856293184d704d/kong/error_handlers.lua#L67) for example. We will provide proper documentation before this is released.